app/assets/javascripts/discourse/app/instance-initializers/subscribe-user-notifications.js

```diff
@@ -30,7 +30,8 @@ export default {
     this.appEvents = owner.lookup("service:app-events");
     this.siteSettings = owner.lookup("service:site-settings");
     this.site = owner.lookup("service:site");
-    this.router = owner.lookup("router:main");
+    // eslint-disable-next-line ember/no-private-routing-service
+    this.router = owner.lookup("main:router");
```

Comment:
Looks like this is passed through to `push-notifications.js`, and then eventually `.handleURL` gets called. Seems like we could use `transitionTo` on the public router service instead?

Reply:
will try it! btw, this line was using `main:router`, not `router:main`! 🤯